Makefile

```diff
@@ -4,21 +4,27 @@ ifeq ($(ARCH),)
 ARCH=$(shell go env GOARCH)
 endif
 
+BUILD_META=-build$(shell date +%Y%m%d)
 ORG ?= rancher
-TAG ?= v3.13.3
+TAG ?= v3.13.3$(BUILD_META)
 
 ifneq ($(DRONE_TAG),)
 TAG := $(DRONE_TAG)
 endif
 
+ifeq (,$(filter %$(BUILD_META),$(TAG)))
+$(error TAG needs to end with build metadata: $(BUILD_META))
+endif
+
 CNI_PLUGINS_VERSION ?= v0.8.7
 
 .PHONY: image-build
 image-build:
 	docker build \
+		--pull \
 		--build-arg ARCH=$(ARCH) \
 		--build-arg CNI_PLUGINS_VERSION=$(CNI_PLUGINS_VERSION) \
-		--build-arg TAG=$(TAG) \
+		--build-arg TAG=$(TAG:$(BUILD_META)=) \
```

Comment:
This syntax is unfamiliar to me, is there a doc explaining the trailing **`=`** ?

Reply:
In case it isn't obvious to other folks: https://www.gnu.org/software/make/manual/html_node/Substitution-Refs.html#Substitution-Refs